codecs.go

```diff
@@ -19,12 +19,13 @@ type Codec interface {
 	// Missing shares must be nil. Returns original + parity data.
 	Decode(data [][]byte) ([][]byte, error)
 	// MaxChunks returns the max number of chunks this codec supports in a 2D
-	// original data square.
+	// original data square. Chunk is a synonym of share.
 	MaxChunks() int
```

Comment:
[suggestion] How about introducing a new method named `MaxShares()` and advising its use as a preferable alternative to` MaxChunks`? The same for `ValidateChunckSize()` -> `ValidateShareSize()`

Reply:
I thought of this but I think it's breaking to add them to the existing interface

Reply:
It would a breaking change, and if the aim is to escape breaking changes, then the interface should not be touched.